config/marc_liberation.yml

```diff
@@ -2,9 +2,11 @@
 # Start with a '/' for absolute paths. Otherwise we assume this is relative to
 # Rails.root
 data_dir: <%= ENV['BIBDATA_DATA_DIR'] || 'data' %>
+tmpdir: <%= Rails.env.production? ? "/mnt/isilon/ingest_scratch" : Dir.tmpdir %>
```

Comment:
Should this be an ENV variable setting instead of hard-coded into the YAML file?

Reply:
Properly it should be but this is a temporary stop-gap and  we'll revisit with #991 so I think it's okay for now.

Reply:
I think for now we can open an issue for the var that @eliotjordan mentioned. and merge this